HPXMLtoOpenStudio/resources/schedules.rb

```diff
@@ -1414,7 +1413,7 @@ def import(schedules_paths)
         end
 
         if max_value_one[col_name]
-          if values.max > 1
+          if values.max > 1.01 || values.max < 0.99 # Allow some imprecision
             fail "Schedule max value for column '#{col_name}' must be 1. [context: #{schedules_path}]"
```

Comment:
Bugfix to match the documentation.

Reply:
Looks like `battery` is not excluded from this check. Should it be? If so, should it have a Units other than "frac" in the docs?

Reply:
Good catch. Just addressed your comment.